sockeye/train.py

```diff
@@ -228,14 +228,23 @@ def main():
         num_embed_source, num_embed_target = args.num_embed
         encoder_num_layers, decoder_num_layers = args.num_layers
 
+        encoder_embed_dropout, decoder_embed_dropout = args.embed_dropout
+        encoder_rnn_dropout, decoder_rnn_dropout = args.rnn_dropout
+        if encoder_embed_dropout > 0 and encoder_rnn_dropout > 0:
+            logger.warning("Strange encoder dropout configuration: "
```

Comment:
thanks for adding the error message. I'm not sure it would be clear to the user what's going on though. Maybe: "WARNING: Turning both the embedding dropout and the RNN input dropout on will lead to two dropout layers on top of each other. Are you sure you want this?" 
Note sure it's clearer :P